load.php

```diff
@@ -300,11 +300,32 @@ function perflab_maybe_set_object_cache_dropin() {
 	}
 
 	if ( $wp_filesystem || WP_Filesystem() ) {
-		// If there is an actual object-cache.php file, rename it.
+		// If there is an actual object-cache.php file, rename it to effectively
+		// back it up.
 		// The Performance Lab object-cache.php will still load it, so the
 		// behavior does not change.
 		if ( $wp_filesystem->exists( WP_CONTENT_DIR . '/object-cache.php' ) ) {
-			$wp_filesystem->move( WP_CONTENT_DIR . '/object-cache.php', WP_CONTENT_DIR . '/object-cache-plst-orig.php' );
+			// If even the backup file already exists, we should not do anything,
+			// except for the case where that file is the same as the main
+			// object-cache.php file. This can happen if another plugin is
+			// aggressively trying to re-add its own object-cache.php file,
+			// ignoring that the Performance Lab object-cache.php file still
+			// loads that other file. This is also outlined in the bug
+			// https://github.com/WordPress/performance/issues/612).
+			// In that case we can simply delete the main file since it is
+			// already backed up.
+			if ( $wp_filesystem->exists( WP_CONTENT_DIR . '/object-cache-plst-orig.php' ) ) {
```

Comment:
What's the `plst` reference?

Reply:
Just a shorthand for "Performance Lab Server Timing" to make the file name more "unique" (e.g. in case another plugin would already do something similar using `-orig` suffix).